db-services/db-robot-store-service/src/main/java/com/qreal/wmp/db/robot/exceptions/translation/ExceptionTranslation.java

```diff
@@ -0,0 +1,48 @@
+package com.qreal.wmp.db.robot.exceptions.translation;
+
+import com.qreal.wmp.db.robot.client.users.UserServiceImpl;
+import com.qreal.wmp.db.robot.exceptions.AbortedException;
+import com.qreal.wmp.db.robot.exceptions.ErrorConnectionException;
+import com.qreal.wmp.db.robot.exceptions.NotFoundException;
+import com.qreal.wmp.thrift.gen.TAborted;
+import com.qreal.wmp.thrift.gen.TErrorConnection;
+import com.qreal.wmp.thrift.gen.TIdNotDefined;
+import com.qreal.wmp.thrift.gen.TNotFound;
+import org.apache.thrift.transport.TTransportException;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+@Aspect
```

Comment:
I guess it would be better to add a comment here describing all this aspect stuff.